src/js/map/maps.js

```diff
@@ -98,26 +100,48 @@ export class MapControl extends Observable {
         this.myEventForwarder.enable();
     }
 
+    async loadSubindicatorData(geo) {
+        this.triggerEvent("map.choropleth.loading");
+
+        let response;
+        let savedObj = this.choroplethData.filter((x) => {
+            return x.geography === geo
+        })[0];
+
+        if (savedObj !== null && typeof savedObj !== 'undefined') {
+            response = savedObj.data;
+        } else {
+            response = await this.api.loadChoroplethData(this.config.profile, geo, 1);
```

Comment:
why is this still `1`? shouldn't this be the indicator?

Reply:
I had left it for later, now instead of `1`, `indicatorId` is sent